capstone/src/main/java/com/google/univiz/CollegeDataApi.java

```diff
@@ -0,0 +1,13 @@
+package com.google.univiz;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * The CollegeDataApi interface is responsible for all REST requests and JSON parsing to the
+ * CollegeData class.
+ */
+public interface CollegeDataApi {
+
+  List<CollegeData> getCollegesById(Set<CollegeId> ids);
```

Comment:
Did we already merge CollegeId into master? If not, merging the repo as is will break future builds unnecessarily.